client/src/utils/API.js

```diff
@@ -12,14 +11,13 @@ export default {
     return axios.post("/api/createjobs", jobData);
   },
 
-  // TODO CHILD & CHILDREN Section
-
-  getAddChild: function(childrenData) {
-    return axios.post("/api/addchild", childrenData);
+  createChild: function(childrenData) {
+    return axios.post("/api/createChild", childrenData);
   },
 
-  deleteAddChild: function(id) {
-    return axios.delete("/api/removeChild/" + id);
+  //parameter: String
+  removeChild: function(childrenData) {
+    return axios.delete(`/api/removeChild/${childrenData}`);
```

Comment:
what happens when you get an empty string?

Reply:
we've added toast notifications